src/Symfony/Component/DependencyInjection/Dumper/PhpDumper.php

```diff
@@ -1332,8 +1336,6 @@ private function getDefinitionsFromArguments(array $arguments)
         foreach ($arguments as $argument) {
             if (is_array($argument)) {
                 $definitions = array_merge($definitions, $this->getDefinitionsFromArguments($argument));
-            } elseif ($argument instanceof ArgumentInterface) {
-                $definitions = array_merge($definitions, $this->getDefinitionsFromArguments($argument->getValues()));
```

Comment:
this is a bug, silent for now because we don't have inline definitions in `$argument->getValues()`, but still a logical mistake